src/explore-education-statistics-admin/src/routes/releaseRoutes.ts

```diff
@@ -123,6 +129,13 @@ export const releaseApiDataSetLocationsMappingRoute: ReleaseRouteProps = {
   protectionAction: permissions => permissions.isBauUser,
 };
 
+export const releaseApiDataSetPreviewRoute: ReleaseRouteProps = {
+  path: '/publication/:publicationId/release/:releaseId/api-data-sets/:dataSetId/preview/:previewTokenId?',
+  title: 'API data set locations mapping',
+  component: ReleaseApiDataSetPreviewTokenPage,
+  protectionAction: permissions => permissions.isBauUser,
+};
```

Comment:
As per earlier comment - would split out routes for:

- initial preview landing page at `/preview` 
- page for viewing a preview token at `/preview/:previewTokenId`